netci.groovy

```diff
@@ -58,13 +58,20 @@ static void addWrappers(def myJob) {
 static void addArtifactArchiving(def myJob, String patternString, String excludeString) {
   myJob.with {
     publishers {
-      archiveArtifacts {
```

Comment:
Yes.   I ran this by @mmitche  to verify that, and he said that was appropriate.

However, from looking at the jenkins DSL, it looks like "publishers" should actually nest under "conditionalAction" (though they can be siblings, it's just deprecated).